databases.go

```diff
@@ -653,6 +656,13 @@ type MySQLConfig struct {
 	BackupHour                   *int     `json:"backup_hour,omitempty"`
 	BackupMinute                 *int     `json:"backup_minute,omitempty"`
 	BinlogRetentionPeriod        *int     `json:"binlog_retention_period,omitempty"`
+	InnodbChangeBufferMaxSize    *uint32  `json:"innodb_change_buffer_max_size,omitempty"`
```

Comment:
```suggestion
	InnodbChangeBufferMaxSize    *int  `json:"innodb_change_buffer_max_size,omitempty"`
```
According to the [API documentation](https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_get_config), its range is `[0 .. 50]`. Perhaps using an `*int` would be sufficient?

Reply:
@loosla ,thank you for the review,
I feel sticking with `*uint32` for `InnodbChangeBufferMaxSize` is a better choice for several reasons. First, the defined range of the value is between 0 and 50, which fits perfectly within the capacity of an unsigned 32-bit integer (`uint32`), ensuring the value is always non-negative. Using `*int` would introduce the potential for negative values, which is unnecessary and could lead to invalid input. Additionally, although both `*uint32` and `*int` pointers have the same size in memory (on most architectures), `uint32` is more semantically appropriate for a field that should never contain negative values. By using `*uint32`, the type system itself helps enforce the expected range of values, reducing the need for additional validation checks. Therefore, choosing `*uint32` ensures better type safety, clarity, and consistency with the API's documented constraints.

Reply:
> Using int as the type in godo will make it easier to handle these fields in Terraform.
> 
> int is more manageable in the long run since Terraform doesn't support uint64. Without this change, we'd need to perform additional steps to convert the value from a string to uint64 in Terraform.
> It's simpler to add a validation in Terraform (e.g., validation.IntAtLeast(0)) to ensure the min which is not always a zero, but can be 1 or 1024 (net_buffer_length).
> 
> There will be more clarity to keep these fields as numeric rather than strings in Terraform.

Vikas, could you please update data type to `*int` for
- InnodbFlushNeighbors
- InnodbReadIoThreads
- InnodbThreadConcurrency
- InnodbWriteIoThreads
- NetBufferLength

too :pray: ?